backend/.env

```diff
@@ -0,0 +1 @@
+JSON_SECRET=ofyGQgzSZrG8nIJ3GQEZ3JKJM90x9tx3bLwdgbCnKItwXJRN7TsL2ydrktyiJYHh
```

Comment:
This .env file should be removed and `.env` should be added in our .gitignore (in the backend folder)

Reply:
Fixed: Changes are reflected in recent commits